### ISSUE TITLE
Make possible to login through GET

### ISSUE THREAD

**Issue by vporton** [YOURLS/YOURLS#2422]
I want an easy (without entering the password) way to login for users of our backend site.

`https://YYY/admin/?username=user1&password=XXX` produces:

> This page isn’t working
> YYY redirected you too many times.

Please make an easy way to login (for uses already secure, such as logged to a backend).

**Comment by dgw** [comment]
Huh. On my YOURLS install it does log me in if I manually go to `/admin/` after the redirect error. But this sounds like a possible bug. One of us will look into it soon. :smile_cat:

Interesting that putting _invalid_ credentials in the URL works fine (shows the login page with an error).

**Comment by PopVeKind** [comment]
Putting a password in a url is highly discouraged for many security reasons.  Users with passwords may save their passwords in their browsers, or you can build an api if you have a way to securely identify a user.

**Comment by dgw** [comment]
Yes, it is discouraged. But YOURLS does support it, or seems to. It should either be removed so the feature does not work at all, or fixed so the feature works consistently.

**Comment by dgw** [comment]
New issue #2424 to remind me (or someone else, if I take too long :grinning:) about fixing the sloppy coding.
